core/ws_comment_events.js

```diff
@@ -215,6 +215,14 @@ Blockly.Events.CommentCreate.prototype.fromJson = function(json) {
  * @param {boolean} forward True if run forward, false if run backward (undo).
  */
 Blockly.Events.CommentCreate.prototype.run = function(forward) {
+  Blockly.Events.CommentCreateDeleteHelper(forward);
+};
+
+/**
+ * Helper function for Comment[Create|Delete].prototype.run
+ * @param {boolean} forward True if run forward, false if run backward (undo).
```

Comment:
I wonder if `create` or similar might be a better name than `forward` for the help that is used in both create and delete cases.

Reply:
+1

Reply:
Sure. Ill change this to create